index.html

```diff
@@ -1,12 +1,193 @@
+            </article>
+            <article>
+                <h2>Don't miss this latest article</h2>
+            </article>
+        </div>
+        <div class="more">
+        </div>
+        <div class="featured">
+        </div>
+        <div class="latest-post">
+            <article>
+                <h2>Don't miss this latest article</h2>
+            </article>
+            <article>
+                <h2>Don't miss this latest article</h2>
+            </article>
+            <article>
+                <h2>Don't miss this latest article</h2>
+            </article>
+        </div>
+        <div class="members">
+            <div></div>
+            <div class="smash-members"></div>
+        </div>
+        <div class="more">
+            <h3>Don't miss this latest article</h3>
+        </div>
+    </main>
+    <footer>
+        <div class="topics">
+            <ul>
+                <li><a href="#">Accessibility</a></li>
+                <li><a href="#">Android</a></li>
+                <li><a href="#">Animation</a></li>
+                <li><a href="#">Apps</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Design</a></li>
+                <li><a href="#">Design Patterns</a></li>
+                <li><a href="#">Design Systems</a></li>
+                <li><a href="#">E-Commerce</a></li>
+                <li><a href="#">Freebies</a></li>
+                <li><a href="#">Graphics</a></li>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Illustrator</a></li>
+                <li><a href="#">Inspiration</a></li>
+                <li><a href="#">iOS</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">Mobile</a></li>
+                <li><a href="#">Pattern Libraries</a></li>
+                <li><a href="#">Performance</a></li>
+                <li><a href="#">Photoshop</a></li>
+                <li><a href="#">Plugins</a></li>
+                <li><a href="#">React</a></li>
+                <li><a href="#">Responsive Web Design</a></li>
+                <li><a href="#">Service Workers</a></li>
+                <li><a href="#">Sketch</a></li>
+                <li><a href="#">Typography</a></li>
+                <li><a href="#">UI</a></li>
+                <li><a href="#">Usability</a></li>
+                <li><a href="#">User Experience</a></li>
+                <li><a href="#">Wallpapers</a></li>
+                <li><a href="#">Web Design</a></li>
+                <li><a href="#">WordPress</a></li>
+                <li><a href="#">Workflow</a></li>
+            </ul>
+        </div>
+        <div class="links">
+            <ul>
+
+                <li><a href="#">One this</a></li>
+                <li><a href="#"> link</a></li>
+                <li><a href="#">to have</a></li>
+                <li><a href="#">access to</a></li>
+                <li><a href="#">the best</a></li>
+                <li><a href="#">posts</a></li>
```

Comment:
![Screenshot from 2019-10-02 19-52-28](https://user-images.githubusercontent.com/19773024/66064631-6f5bae80-e54e-11e9-8b68-e51415b51af1.png)


These links still don't appear. Please make them visible and properly centered.